### ISSUE TITLE
Feature Request: Text Wrapping

### ISSUE THREAD

**Issue by stuartmscott** [fyne-io/terminal#8]
When running on Ubuntu long lines get truncated, ideally the line would be wrapped instead.

If it is desirable to have both behaviours then perhaps this could be a user preference. Additionally when configured to truncate, there should be a shortcut to scroll horizontally.

**Comment by stuartmscott** [comment]
Comparison with system default;
![FyneTerm3](https://user-images.githubusercontent.com/1117080/114937923-4a0e5880-9df3-11eb-86be-329280099b83.png)

**Comment by GuoRanYiMao** [comment]
> l [on 2 Jul 2021](#event-4972047440)
hi 
Have you solved this problem?

**Comment by andydotxyz** [comment]
> Have you solved this problem?

No, when the feature is added we will mark this issue as resolved.
